requirements.txt

```diff
@@ -5,6 +5,7 @@ click-didyoumean>=0.3.0
 datasets>=2.18.0
 gguf>=0.6.0
 GitPython>=3.1.42
+sigstore>=3.0.0
```

Comment:
Hi! Are you aware of this library here: https://github.com/sigstore/model-transparency/

It may be a better library to use for signing models that are split into multiple tensor files and have possibly other files laying around that may be worth signing or verying. This library would sign all the files and files found in directories under a `model_path`: https://github.com/sigstore/model-transparency/blob/13459e976001d892f254f8e9927a092461d47ed8/src/sign.py#L191-L197

My feeling is that you're getting into an area of what will be the industry-accepted way of signing models and I think the model signing library may be better.

Reply:
@stefanberger Thanks for the review (and hello again, long time no see)!

> Are you aware of this library here:

Yes! I've been involved in the Sigstore model-signing SIG since before I wrote this PR - I left it in draft format with the intention of reworking it using the model-signing library once it hit stable. That has not yet happened, but when it does, I might revisit.

> My feeling is that you're getting into an area of what will be the industry-accepted way of signing models and I think the model signing library may be better.

Totally agreed! I wrote this PR mostly as a very quick and dirty proof of concept of model signing - I wasn't intending to ship it in this state.

Reply:
> Yes! I've been involved in the Sigstore model-signing SIG since before I wrote this PR - I left it in draft format with the intention of reworking it using the model-signing library once it hit stable.

Do you know what it will take for it to 'hit stable'?